infrastructures/kubernetes/src/main/java/org/eclipse/che/workspace/infrastructure/kubernetes/namespace/KubernetesDeployments.java

```diff
@@ -878,13 +862,33 @@ private String getPodName(String name) throws InfrastructureException {
             .list()
             .getItems();
     if (pods.isEmpty()) {
-      throw new InfrastructureException(String.format("Failed to find pod with name %s", name));
+      return Optional.empty();
     } else if (pods.size() > 1) {
       throw new InfrastructureException(
-          String.format("Found multiple pods in DeploymentConfig %s", name));
+          String.format("Found multiple pods in DeploymentConfig '%s'", name));
     }
 
-    return pods.get(0).getMetadata().getName();
+    return Optional.of(pods.get(0));
+  }
+
+  /**
+   * Returns the name of a specified Pod given either the actual Pod name or the name of the
+   * DeploymentConfig that controls it. <br>
+   * This is necessary because we are trying to transparently wrap Pods in DeploymentConfigs;
+   * attempting to create a Pod named {@code testPod} will result in a DeploymentConfig with name
+   * {@code testPod}, which will in turn create a Pod named e.g {@code testPod-1-xxxxx}.
+   *
+   * @param name Pod or DeploymentConfig name
+   * @return the name of the intended pod.
+   * @see
```

Comment:
complete or remove the see tag

Reply:
Good catch. This line was added some time ago and I just moved it. I have no idea what should be referenced here, so removed.